lib/wegift/models/remote_code.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'uri'
+
+module Wegift
+  class RemoteCode < Response
+    attr_reader :url
+
+    # response/success
+    attr_reader :amount, :barcode_format, :barcode_string, :code, :expiry_date,
+      :pin, :type
+
+    def initialize(url:)
+      @url = url
+    end
+
+    def self.get(url)
+      uri = URI(url)
+      conn = Faraday.new(url: "#{uri.scheme}://#{uri.host}")
+
+      response =  conn.get("#{uri.path}?format=json") do |req|
```

Comment:
Is this some weird requirement on their side? So the Content-Type header is ignored? Or do they actually need both?

Reply:
I think they are trying to shoe-horn an existing endpoint (public remote voucher) and added the json version of it later.

Reply:
So this is required or not?

Reply:
no required

Reply:
sorry, didn't understand your first comment. `Content-Type` is not required but for the sake of validity, we're adding it to the request.

Reply:
I guess the header is not needed than?